stubs/docker-compose.yml

```diff
@@ -12,6 +12,8 @@ services:
             - '${APP_PORT:-80}:80'
```

Comment:
To make it work, I had to put
```
XDEBUG: '${SAIL_XDEBUG:-false}'
XDEBUG_PORT: '${SAIL_XDEBUG_PORT:-9000}'
```
under the `services:laravel.test:build:args` section instead of `services:laravel.test:environment`